ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryGuid.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import java.nio.charset.StandardCharsets;
+import java.util.UUID;
+import org.joda.time.DateTime;
```

Comment:
Is that the DateTime library that we use throughout?

Reply:
You're right. I important the more frequently used one.